web/src/input_pill.ts

```diff
@@ -478,7 +479,10 @@ export function create<ItemType extends {type: string}>(
         store.$parent.on("copy", ".pill", function (this: HTMLElement, e) {
             const {item} = funcs.getByElement(this)!;
             assert(e.originalEvent instanceof ClipboardEvent);
-            e.originalEvent.clipboardData?.setData("text/plain", store.get_text_from_item(item));
+            e.originalEvent.clipboardData?.setData(
+                "text/plain",
+                store.get_display_value_from_item(item),
```

Comment:
I guess this will change the behavior for other pills, which I am not sure we want.